roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -98,16 +98,5 @@
         name: NetworkManager
         state: reloaded
```

Comment:
I can still see 'Enable openvswitch service if installed' and other removed tasks in aee jobs logs on my test env. I haven't tested this change properly, sorry for the noise